README.md

```diff
@@ -90,7 +94,89 @@ To run this example you will need following extra dependency
 ```elixir
 {:membrane_http_adaptive_stream_plugin, github: "membraneframework/membrane_http_adaptive_stream_plugin"}
 ```
+### Client
+Example Streaming Client pipeline might look like this:
+```elixir
+defmodule Example.Stream do
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(options) do
+    children = [
+      video_source: %Membrane.File.Source{location: options[:video_file_path]},
+      video_parser: %Membrane.H264.FFmpeg.Parser{
+        framerate: {25, 1},
+        alignment: :au,
+        attach_nalus?: true,
+        skip_until_keyframe?: true
+      },
+      audio_parser: %Membrane.AAC.Parser{
+        out_encapsulation: :none
+      },
+      audio_source: %Membrane.File.Source{location: options[:audio_file_path]},
+      video_realtimer: Membrane.Realtimer,
+      audio_realtimer: Membrane.Realtimer,
+      video_payloader: Membrane.MP4.Payloader.H264,
+      rtmps_sink: %Membrane.RTMP.Sink{rtmp_url: options[:rtmp_url]}
+    ]
+
+    links = [
+      link(:video_source)
+      |> to(:video_parser)
+      |> to(:video_realtimer)
+      |> to(:video_payloader)
+      |> via_in(:video)
+      |> to(:rtmps_sink),
+      link(:audio_source)
+      |> to(:audio_parser)
+      |> to(:audio_realtimer)
+      |> via_in(:audio)
+      |> to(:rtmps_sink)
+    ]
+
+    {{:ok, spec: %ParentSpec{children: children, links: links}}, %{}}
+  end
+
+  @impl true
+  def handle_element_end_of_stream({:rtmps_sink, pad}, _ctx, %{finished_streams: [closed_pad]} = state) do
+    Membrane.Pipeline.stop_and_terminate(self())
+    {:ok, Map.put(state, :finished_streams, [pad, closed_pad])}
+  end
+
+  @impl true
+  def handle_element_end_of_stream({:rtmps_sink, pad}, _ctx, state) do
+    {:ok, Map.put(state, :finished_streams, [pad])}
+  end
 
+  @impl true
+  def handle_element_end_of_stream(_element, _ctx, state) do
+    {:ok, state}
+  end
+end
+```
+It will connect to RTMP server provided via URL and stream H264 video and AAC audio muxed to FLV format.
+RTMP server that will receive this stream can be launched with ffmpeg by running the following commands:
+```bash
+export RTMP_URL=rtmp://localhost:1935
+ffmpeg -listen 1 -f flv -i rtmp://localhost:1935 -c copy dest.flv
+```
+It will receive stream and once streaming is completed dump it to .flv file.
+Run it with:
+```elixir
+pipeline_options = %{
+  video_file_path: "bun33s_480x270.h264",
+  audio_file_path: "bun33s.aac",
+  rtmp_url: System.get_env("RTMP_URL")
+}
```

Comment:
Options for an example pipeline are overkill. Especially when they're passed in different ways.